src/Item_Project.php

```diff
@@ -265,16 +269,97 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
+                    self::showForAsset($item);
+                }
+                break;
         }
         return true;
     }
+
+    private static function showForAsset(CommonDBTM $item): void
+    {
+
+        $item_project = new self();
+        $item_projects = $item_project->find([
+            'itemtype' => $item->getType(),
+            'items_id' => $item->getID()
+        ]);
+
+        $used = $entries = [];
+
+        foreach ($item_projects as $value) {
+            $used[] = $value['projects_id'];
+            $project = new Project();
+            $result = $project->getFromDB($value['projects_id']);
+
+            if ($result === false) {
+                continue;
+            }
+
+            $priority = CommonITILObject::getPriorityName($project->fields['priority']);
+            $prioritycolor  = $_SESSION["glpipriority_" . $project->fields['priority']];
+            $state = ProjectState::getById($project->fields['projectstates_id']);
+
+            $data = [
+                'name' => $project->getLink(),
+                'projectstates_id' => $state !== false
+                    ? [
+                        'content' => $state->fields['name'],
+                        'color' => $state->fields['color']
+                    ] : '',
+                'priority' => [
+                    'content' => $priority,
+                    'color' => $prioritycolor
+                ],
+                'percent_done' => Html::getProgressBar((float)$project->fields['percent_done'])
```

Comment:
Not a big deal, but should we display these information if the user has no right to see this specific project (`!$project->can($project->fields['id'], READ)`)?